src/widgets/merch-catalog/ui/pagination/pagination.tsx

```diff
@@ -0,0 +1,81 @@
+'use client';
+import { faChevronLeft, faChevronRight } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import classNames from 'classnames/bind';
+
+import styles from './pagination.module.scss';
+
+const cx = classNames.bind(styles);
+
+export type PaginationProps = {
+  currentPage: number;
+  totalPages: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination = ({ currentPage, totalPages, onPageChange }: PaginationProps) => {
+  const getPaginationItems = () => {
+    const leftCount = 3;
+    const rightCount = 2;
```

Comment:
I suggest making the variable names more understandable
```suggestion
    const visiblePagesInLeftSection  = 3;
    const visiblePagesInRightSection = 2;
```

Reply:
Done